src/backend/src/health/health.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Get, HttpStatus } from '@nestjs/common';
+import {
+  HealthCheckService,
+  HttpHealthIndicator,
+  HealthCheck,
+} from '@nestjs/terminus';
+import { ApiExcludeEndpoint } from '@nestjs/swagger';
+@Controller('health')
+export class HealthController {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  constructor() {}
```

Comment:
I strongly urge not to disable eslint unless unavoidable If it is an empty constructor, I guess you can get rid of it.

Reply:
but this is an empty constructor, that's why I add it

Reply:
I mean - you can get rid of the constructor itself.